src/main/java/seedu/address/model/person/Person.java

```diff
@@ -28,6 +28,8 @@ public class Person {
     private final Address address;
     private final GroupTagSet groupTags = new GroupTagSet();
     private final ModuleTagSet moduleTags = new ModuleTagSet();
+    private final LessonSet lessons = new LessonSet();
+    // to change constructor for person????
```

Comment:
I think if it is a part of ModuleTagSet, it should be fine. That way there’s fewer things to change for Person constructor.